src/Travis.jl

```diff
@@ -0,0 +1,125 @@
+export Travis
+
+"""
+Package functions for interacting with Travis.
+
+$(EXPORTS)
+"""
+module Travis
+
+using Compat, DocStringExtensions
+import JSON
+import ..Documenter:
+    Utilities,
+    GitHub
+
+export genkeys
+
+test_for(program) = if !success(`which $program`)
+    error("$program not found.")
+end
+
+which() = foreach(test_for, ("which", "git", "ssh-keygen", "travis") )
+
+windows_prefix() = if is_windows()
```

Comment:
I try to avoid `?` because it messes up coverage stats. Though it doesn't matter too much cause this part of the code isn't currently tested.